### ISSUE TITLE
[api-extractor] Release tags aren't respected for const exports

### ISSUE THREAD

**Issue by nvh** [Microsoft/web-build-tools#936]
## Reproduction
Create a `export const variableDeclaration` and add a `@beta` tag to it

## Expected
variableDeclaration won't show up in the public rollup `.d.ts` file

## Observed
It does.

## Remarks
There is an example of this in the test files in this repo:

The variable is defined here as `@beta`
https://github.com/Microsoft/web-build-tools/blob/master/build-tests/api-extractor-test-04/src/index.ts#L26

But shows up in the public rollup file here:
https://github.com/Microsoft/web-build-tools/blob/master/build-tests/api-extractor-test-04/dist/public/api-extractor-test-04.d.ts#L53

**Comment by nvh** [comment]
Some more investigation:
```typescript
/**
 * @internal
 */
export const variableDeclaration1: string = "hello"
export /** @internal */ const variableDeclaration2: string = "hello"
export const /** @internal */ variableDeclaration3: string = "hello"
```
gets translated into 
```typescript
declare const variableDeclaration1: string;

declare const variableDeclaration2: string;

declare const /** @internal */ variableDeclaration3: string;
```

**Comment by pgonzal** [comment]
So as I mentioned [in Gitter](https://gitter.im/web-build-tools/web-build-tools?at=5bdc8b4eee8e69215343aeb5), the variable declaration syntax is tricky in general because you can write something like this:

> ```ts
> export const a = 1, b = 2, [c, d] = [3, 4];
> ```
> Where would you expect to find the doc comment for "b" or "c" in this expression?

Based on your findings above, I'm thinking it's pretty reasonable for API Extractor to ask that exported variable declarations must be separated into individual statements.  So the above line would need to be rewritten in this equivalent form, if you want API Extractor to analyze it:

```ts
/** docs */
export const a = 1;
/** docs */
export const b = 2;
/** docs */
export const c = 3;
/** docs */
export const d = 4;
```

With that change, the .d.ts rollup fix should be pretty straightforward.  In [DtsRollupGenerator.ts](https://github.com/Microsoft/web-build-tools/blob/5e289f8d890c97aa5e8eadc7c34e04dab9105161/apps/api-extractor/src/generators/dtsRollup/DtsRollupGenerator.ts#L460) we would need to check for `ts.SyntaxKind.VariableDeclaration` or whatever and find the right node to pass to `getJSDocCommentRanges()`.  There is already something similar happening in [_modifySpan()](https://github.com/Microsoft/web-build-tools/blob/5e289f8d890c97aa5e8eadc7c34e04dab9105161/apps/api-extractor/src/generators/dtsRollup/DtsRollupGenerator.ts#L341).

**Comment by pgonzal** [comment]
I will fix this.  I'm encountering the same issue in my API Extractor 7 PR branch, which now supports API reports for variable declarations.  I'll see if I can also backport it to AE6.

**Comment by pgonzal** [comment]
@nvh this should be fixed in API Extractor 6.1.5
